scripts/generate_release_updates.py

```diff
@@ -0,0 +1,136 @@
+
+import os
+import re
+
+import feconf
+import python_utils
+
+from . import common
+
+SECTIONS_TO_ADD = [
+    '[Add main changes]',
+    '[Add editorials/announcements if required]',
+    '[Add Author details (Use Email C&P Blurbs about authors from '
+    'release_summary.md)]',
+    '[Add names of release testers]',
+    '[Add name of QA team lead for the release]',
+    '[Add your name]']
+RELEASE_MAIL_MESSAGE_TEMPLATE = (
+    'Hi all,\n\n'
+    '   The main changes in this release are %s.\n'
+    '   %s.\n'
+    '   %s\n'
+    '   Finally, I\'d like to thank %s for their help with pre-release '
+    'testing, bug-fixing and QA, as well as %s for leading the QA team for '
+    'this release.\n\n'
+    'Thanks,\n'
+    '%s\n') % tuple(SECTIONS_TO_ADD)
+
+PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+RELEASE_MAIL_MESSAGE_FILEPATH = os.path.join(
+    PARENT_DIR, 'release_mail_message.txt')
+
+
+def write_message_template():
+    """Adds the template message to release mail filepath."""
+    with python_utils.open_file(RELEASE_MAIL_MESSAGE_FILEPATH, 'w') as f:
+        f.write(RELEASE_MAIL_MESSAGE_TEMPLATE)
+
+    common.ask_user_to_confirm(
+        'Please make updates to following file %s for generating the '
+        'release announcement mail by adding:\n'
+        '   1. Main changes for release\n'
+        '   2. Editorials/announcements if required\n'
+        '   3. Author details from release_summary.md\n'
+        '   4. Names of release testers\n'
+        '   5. Name of QA Team lead\n'
+        '   6. Your name\n' % RELEASE_MAIL_MESSAGE_FILEPATH)
+
+
+def check_updated_message():
+    """Checks the message after the mail template is updated by the
+    user.
+
+    Raises:
+        Exception: If the message still contains sections from template
+            which are not updated.
+    """
+    with python_utils.open_file(RELEASE_MAIL_MESSAGE_FILEPATH, 'r') as f:
+        release_mail_message = f.read()
+        extra_sections = [
+            section for section in SECTIONS_TO_ADD if (
+                section in release_mail_message)]
+        if extra_sections:
+            raise Exception(
+                'Template not formatted correctly. '
+                'Following sections still not updated: %s.\n'
+                'Please re-run the scripts and make the updates again.' % (
+                    ', '.join(extra_sections)))
+
+
+def send_and_check_updates():
+    """Asks the user to send release announcement mail and check if
+    it is in announcement category.
+    """
+    common.open_new_tab_in_browser_if_possible(
+        'https://www.gmail.com')
+    common.ask_user_to_confirm(
+        'Please copy the mail message from %s and send the email to:\n'
+        '   TO: oppia-dev@googlegroups.com\n'
+        '   BCC: oppia@googlegroups.com\n'
+        '   BCC: oppia-announce@googlegroups.com\n'
+        '   BCC: each new contributor for this release\n' % (
```

Comment:
Do you want to generate this list directly for them, so they don't need to manually extract the emails again?

Reply:
I am sorry but I did not understand the meaning of generating list directly. Do you want me to print two separate lists in new lines for to and bcc?

Reply:
Yes, so should I generate the email addresses for new contributors as well from the release summary file in that list and add a warning message for user to double check them before copy pasting them?

Reply:
Yep, that's the idea.